coalib/parsing/LineParser.py

```diff
@@ -30,7 +30,9 @@ def __init__(self,
                                             "section" despite of the current
                                             section.
         """
-        section_name_surroundings = section_name_surroundings or {"[": "]"}
+        section_name_surroundings = (
+            {"[": "]"} if section_name_surroundings is None
+            else section_name_surroundings)
```

Comment:
@Makman2 I did not change this to `MappingProxyType` because on line 53 of `test_section_name_parsing` function in `tests/parsing/LineParser`, we are adding a new key to default dictionary and `MappingProxyType` does not allow as it is immutable.